### PR TITLE
[GES] Fix typos and improve formatting

### PR DESCRIPTION
**Pull request by Pesteves2002**

### CODE REVIEW

File: content/ges/0002-organizacoes-recursos-humanos.md [hunk 1]
```diff
@@ -39,20 +39,20 @@ Cada empresa/ organização é influenciada por:
 - **Responsabilidade Social**: Responsabilidade dos gestores em fazerem escolhas e promoverem ações que contribuem para o bem-estar e interesses da sociedade e da organização. Numa definição menos formal, podemos dizer que caem no âmbito da responsabilidade social decisões que vão para além do previsto e estipulado, por exemplo, questões ambientais, atividades em benefício da sociedade, respeito para os consumidores.
 
 :::info[Exemplo]
-O IST preza-se pela sua excelência, pela formação teórica e prática de elevado nível dos seus alunos (**Elemento invisel da cultura**),
-tem como sua marca o logótipo que desde o início inclui o símbolo da integral e as cores azul e branco (**Elemento Visivel de Cultura**).
-Esta instituição é considerada bastante séria e assume uma posição antipraxe (**Código moral - Ética**).
+O IST preza-se pela sua excelência, pela formação teórica e prática de elevado nível dos seus alunos (**Elemento Invisíveis da Cultura**),
```
Comment (diogotcorreia) [inline content/ges/0002-organizacoes-recursos-humanos.md:42]: > Elemento Invisíveis

:thinking:

File: content/ges/0002-organizacoes-recursos-humanos.md [hunk 2]
```diff
@@ -129,64 +129,64 @@ Presidente da República > Primeiro Ministro > Ministros > Secretários > etc.
...
      como especialistas em formação e engenheiros, para se focar na sua atividade principal.
 
-   ![Rede Virtual](./assets/0002-rede-virtual.png)
+   ![Rede Virtual](./assets/0002-rede-virtual.png#dark=1)
 
 ## Gestão de Recursos Humanos
 
 A gestão de recursos humanos inclui a concepção e implementação de sistemas formais que permitam
-a utilização do talento e capacidades das pessoas, de forma eficiente e eficaz, para atingir as metas da organização.
+a utilização do talento e capacidades das pessoas, de forma eficiente e eficaz, para atingir as metas da organização.\
```
Comment (diogotcorreia) [inline content/ges/0002-organizacoes-recursos-humanos.md:137]:
```suggestion
a utilização do talento e capacidades das pessoas, de forma eficiente e eficaz, para atingir as metas da organização.  
```

Prefer double space over backslash for line break on markdown

File: content/ges/0002-organizacoes-recursos-humanos.md [hunk 3]
```diff
@@ -129,64 +129,64 @@ Presidente da República > Primeiro Ministro > Ministros > Secretários > etc.
...
 
-- **Formação interna/externa**: A existência de aprendizagem em aulas, formação dos trabalhadores de modo a que estes possam evoluir na sua carreira.
+- **Formação Interna/Externa**: A existência de aprendizagem em aulas, formação dos trabalhadores de modo a que estes possam evoluir na sua carreira.
 
-- **Avaliação de desempenho**: O processo de observação e valorização do desempenho dos empregados; registo do desempenho;
-  informação dos avaliados e aos gestores responsáveis na tomade de decisões com base nos desempenhos dos individuos.
-  O sistemas de avaliação devem ser objetivos/claros, adaptados às mudanças das condições e dos requisitos para o exercicio de tarefas.
+- **Avaliação de Desempenho**: O processo de observação e valorização do desempenho dos empregados; registo do desempenho;
+  informação dos avaliados e aos gestores responsáveis na tomade de decisões com base nos desempenhos dos individuos.\
+  Os sistemas de avaliação devem ser objetivos/claros, adaptados às mudanças das condições e dos requisitos para o exercicio de tarefas.\
```
Comment (diogotcorreia) [inline content/ges/0002-organizacoes-recursos-humanos.md:191]:
```suggestion
  informação dos avaliados e aos gestores responsáveis na tomade de decisões com base nos desempenhos dos individuos.  
  Os sistemas de avaliação devem ser objetivos/claros, adaptados às mudanças das condições e dos requisitos para o exercicio de tarefas.  
```

Prefer double space over backslash for line break on markdown

File: content/ges/0003-ambiente-economico.md [hunk 4]
```diff
@@ -26,9 +26,9 @@ A economia subdivide-se em duas grandes áreas:
 
 - **Microeconomia**: Estuda o comportamento económico de entidades individuais como os mercados, as empresas e as famílias.
 
-Como o próprio nome indica a **macroeconomia** retrata o grande cenário - o Estado, a economia Nacional
-e as relações económicas internacionais - recorrendo a indicadores, como o PIB, a Taxa de Desemprego, a inflação, etc.
-A **microeconomia** foca-se no pequeno, no detalhe, nos individuos e nas suas ações, nas empresas e no seu processamento produtivo.
+Como o próprio nome indica a **Macroeconomia** retrata o grande cenário - o Estado, a Economia Nacional
```
Comment (diogotcorreia) [inline content/ges/0003-ambiente-economico.md:29]:
```suggestion
Como o próprio nome indica, a **Macroeconomia** retrata o grande cenário - o Estado, a Economia Nacional
```

File: content/ges/0003-ambiente-economico.md [hunk 5]
```diff
@@ -62,17 +62,17 @@ $$
...
   :::
 
-- **Produtividade do trabalho**: Rácio entre o PIB e o número de horas efetivamente trabalhadas.
+- **Produtividade do Trabalho**: Rácio entre o PIB e o número de horas efetivamente trabalhadas.
   Trata-se de um indicador de competitividade económica, pois elimina-se as diferenças na distribuição de população empregada a tempo completo e parcial.
 
 - **Taxa de Desemprego**: Percentagem de população ativa em situação de desemprego involuntário.
 
-- **Taxa de Inflação**: Taxa de variação de preços medido pelo indice de preços do consumidor (IPC).
+- **Taxa de Inflação**: Taxa de variação de preços medido pelo indíce de preços do consumidor (IPC).
```
Comment (diogotcorreia) [inline content/ges/0003-ambiente-economico.md:75]:
```suggestion
- **Taxa de Inflação**: Taxa de variação de preços medido pelo índice de preços do consumidor (IPC).
```

File: content/ges/0003-ambiente-economico.md [hunk 6]
```diff
@@ -125,12 +125,12 @@ em que $n = \text{número de anos}$
 
 :::
 
-A **deflação** é o processo inverso à inflação, ou seja, uma diminuição do indice de preços do consumidor, associada a uma queda de preços.
+A **Deflação** é o processo inverso à inflação, ou seja, uma diminuição do indíce de preços do consumidor, associada a uma queda de preços.
```
Comment (diogotcorreia) [inline content/ges/0003-ambiente-economico.md:128]:
```suggestion
A **Deflação** é o processo inverso à inflação, ou seja, uma diminuição do índice de preços do consumidor, associada a uma queda de preços.
```

File: content/ges/0004-informacao-financeira.md [hunk 7]
```diff
@@ -115,7 +115,7 @@ $$
 \Delta\text{ativo} = \Delta\text{passivo} + \Delta\text{capital próprio}
 $$
 
-O capital próprio traduz o valor contabilístico da empresa num determinado momento:
+O capital próprio traduz o valor contabilístico da empresa num determinado momento:\
```
Comment (diogotcorreia) [inline content/ges/0004-informacao-financeira.md:118]:
```suggestion
O capital próprio traduz o valor contabilístico da empresa num determinado momento:  
```
Prefer double space over backslash for line break on markdown

File: content/ges/0004-informacao-financeira.md [hunk 8]
```diff
@@ -176,7 +176,7 @@ criaram e iniciaram a atividade de uma empresa de computadores. Nesse mês ocorr
 - Durante o 1.º mês a empresa faturou 15.000€, com pagamento a 60 dias. O material vendido teve um custo de aquisição de 10.000€.
 - Pagou-se no final do mês ordenados e outros encargos com o pessoal no montante de 1.500€
 
-Balança do resultados:
+Balança do Resultados:
```
Comment (diogotcorreia) [inline content/ges/0004-informacao-financeira.md:179]:
```suggestion
Balança dos Resultados:
```

**Review by diogotcorreia — CHANGES_REQUESTED** [review]

**Review by Pesteves2002 — COMMENTED** [review]
NOW

**Review by diogotcorreia — APPROVED** [review]

**Pull request merged**